assets/js/schematics.js

```diff
@@ -7,14 +7,15 @@
             'options': [
                 [
                     'imu',
-                    (imu) => imu == 'bno' ? 'd1_BNO-Main.png' : (imu == 'mpu' ? 'd1_MPU-Main.png' : (imu == 'mpu9250' ? 'd1_MPU9250-Main.png' : 'd1_BMI160-Main.png'))
-                    
+                    (imu) => imu == 'bno' ? 'd1_BNO-Main.png' : (imu == 'mpu' ? 'd1_MPU-Main.png' : (imu == 'mpu9250' ? 'd1_MPU9250-Main.png' : (imu == 'bmi' ? 'd1_BMI160-Main.png' : 'd1_MPU+QMC-Main.png')))
+                    // I don't think this is how this should be done at all...
```

Comment:
You aren't wrong here

Reply:
It's a function. It's okay to not have it in a single line. It's grown way past that, a inline if was okay for two options, not when it expanded to three, and changing to four now looks quite silly ^^.

It's well time to change it to a function that contains a switch or a little inline look up table maybe? :)